### ISSUE TITLE
unregisterHandler missing undefined check

### ISSUE THREAD

**Issue by leolux** [knalli/angular-vertxbus#57]
Version 1.1.3 has the following code to unregister a callback function:
```
 unregisterHandler: function(address, callback) {
          var index;
          if (wrapped.handlers[address]) {
            index = wrapped.handlers[address].indexOf(callback);
            if (index > -1) {
              wrapped.handlers[address].splice(index, 1);
            }
          }
          if (wrapped.handlers[address].length < 1) {
            wrapped.handlers[address] = void 0;
          }
          if (connectionState === vertxEventBus.EventBus.OPEN) {
            return util.unregisterHandler(address, callback);
          }
        }
```
The issue is in the line following line:
```
          if (wrapped.handlers[address].length < 1) {
```
It produces the following error:
TypeError: Cannot read property 'length' of undefined
    at Object.wrapped.unregisterHandler (http://localhost/bower_components/angular-vertxbus/dist/angular-vertxbus.js:666:40)

Currently I can reproduce this error in my app as follows:
1. Open the URL of the app
2. Wait for the app to end itself (unregister works at this point)
3. Click the back button of the browser (here unregister produces the error)

In the last step the instance vertxEventBusService is still available and invoked to unregister the same function again and leading to the error.

**Comment by knalli** [comment]
@leolux  Thank you again. Same issue like last time, only a few lines further. Args.

Fixed in 1.1.4

**Comment by knalli** [comment]
I've also updated the v2 branch to 2.0.0-beta.5
